eng/azure-pipelines.yml

```
@@ -42,7 +42,7 @@ stages:
             name: NetCore1ESPool-Internal
             demands: ImageOverride -equals Build.Ubuntu.1804.Amd64
         container:
-          image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-webassembly3130-20230129084957-f7eb839
+          image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-webassembly3130-20230130145247-f7eb839
```

we're supposed to be using the "latest" docker tags, so just 
```suggestion
          image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-webassembly3130
```

---

this is temporary, so that we could prepare dotnet/runtime PR. once we have everything ready, it will use the `ubuntu-18.04-webassembly` image

---

We'll probably need to keep different images in the future anyway since we're not planning on bumping emscripten in e.g. release/7.0 and the goal is to move to these "latest" tags everywhere

---

We might make our mind meanwhile then and discuss what would work best and change it when the rest of our changes are ready. For now I would like to use these. The `ubuntu-18.04-webassembly3130` image has only datetime tags right now, we can add floating tags later if needed.

When I chose the `webassembly3130` name, I thought about it as temporary. If we want to introduce separate images, then something like webassembly-net8 might work better? The emscripten 3.1.30 might not be the final version yet.

---

Yup `webassembly-net8` sounds better, I agree. The floating tag should be created automatically AFAIK (maybe only once there was a "production" rollout once a week).

---

It is specified in the manifest, which tags are created. So `ubuntu-18.04-webassembly` has the floating tags https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/src/ubuntu/manifest.json#L447-L448, while `ubuntu-18.04-webassembly3130` has not https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/src/ubuntu/manifest.json#L460.

---

ah interesting, thanks